Tools/create-release.bash

```diff
@@ -11,7 +11,12 @@ set -o errexit
 TYPE=$1
 
 release_version=$(./Tools/create-release.py ${TYPE})
+cat AndroidSDKCore/sdk-version.txt
```

Comment:
what would this line do?

Reply:
Are we adding the version in this file somehow?

Reply:
Yes that file is already in to note our current version. It's read during build time to be hard-coded in the binary. This line `cat...` Just displays the version. It's a debug line but I think useful.